htlcswitch/circuit_map.go

```diff
@@ -213,6 +213,10 @@ type CircuitMapConfig struct {
 	// ExtractErrorEncrypter derives the shared secret used to encrypt
 	// errors from the obfuscator's ephemeral public key.
 	ExtractErrorEncrypter hop.ErrorEncrypterExtracter
+
+	// CheckResolutionMsg retrieves a resolution message from the
+	// resolution store with the passed CircuitKey.
+	CheckResolutionMsg func(outKey *CircuitKey) error
```

Comment:
`godoc` states it retrieves the resolution, but it actually just returns an error instead?

Reply:
So maybe this should actually return a `bool`?